src/store/initialStates/userInitialState.js

```diff
@@ -78,5 +78,9 @@ module.exports = {
     tags: ['read', 'create', 'edit', 'delete'],
     users: ['read', 'create', 'edit', 'delete'],
     permissions: ['read', 'create', 'edit', 'delete']
-  }
+  },
+  followers: [],
+  following: [],
+  followUser: [],
```

Comment:
@sengayire, Why do you declare this as an array yet in your code you are using it as a JSON object?